cmd/ris/risserver/server.go

```diff
@@ -251,6 +270,24 @@ func (s *Server) GetRouters(c context.Context, request *pb.GetRoutersRequest) (*
 	return resp, nil
 }
 
+type RequestWithVRF interface {
+	GetVrfId() uint64
+	GetVrf() string
+}
+
+func getVRF(req RequestWithVRF) (uint64, error) {
```

Comment:
This should be called getVRFID(), otherwise it's confusing why this returns an ID and not a string as getVrf about does.